test/js/src/main/scala/zio/test/FileIOPlatformSpecific.scala

```diff
@@ -0,0 +1,91 @@
+package zio.test
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSImport
+import zio.{Task, ZIO}
+
+
+trait FileIOPlatformSpecific {
```

Comment:
We already have operators for working with files in ZIO core for the JVM. Can we create analogous versions of these for Scala.js so we can make this available to users more generally in a platform agnostic way and get this logic out of ZIO Test?

Reply:
Yes, I will move it there.